config/rbac/instascale_role.yaml

```diff
@@ -0,0 +1,102 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  name: instascale-role
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  verbs:
+  - get
+- apiGroups:
+  - ""
+  resources:
+  - nodes
+  verbs:
+  - get
+  - list
+  - patch
+  - update
+- apiGroups:
+  - ""
+  resourceNames:
+  - instascale-ocm-secret
```

Comment:
The OCM Secret can now be referenced in the operator configuration, so it's not hard-coded anymore.